components/director/cmd/ns-adapter/README.md

```diff
@@ -0,0 +1,315 @@
+# NS-Adapter
+
+## Overview
+
+### Basic concepts
+
+SAP Cloud Connector (SCC) - a component which runs on-premise, in customer datacenter, and provides connectivity from the Cloud to on-premise systems.
+
+Notification Service (NS) - a Cloud service in which SCCs are registered so that cloud applications can connect to on-premise systems exposed via these SCCs. An SCC registers in NS in the context of a subaccount. Multiple SCCs may be registered in NS in a single subaccount, and also one SCC may be registered in NS in multiple subaccounts. Only applications running in the subaccount in which the SCC is registered can access the on-premise systems exposed by the SCC.
+
+Full report - NS reports to CMP all SCCs currently registered in it, with all their exposed on-premise systems.
+
+Delta (incremental) report - NS reports to CMP only those SCCs, which changed since the last report.
+
+## Details
+
+This section describes the API schema that a Notification Service must implement to integrate with the NS-Adapter.
+
+### Authorization
+
+Valid JWT token with "tenant.consumerTenant" and "tenant.externalTenant" claims. ConsumerTenant and externalTenant should contain the internal ID and the external ID from existing business_tenant_mappings record in Compass database.
```

Comment:
I think this is not very clear. What's the difference between consumer tenant and external tenant, what do they really correlate to (ex. which one is the one associated with the subaccount in which the SCC is)?

Also how would NS know of internal ID that's in Compass DB?